packages/material/scss/list/_variables.scss

```diff
@@ -3,16 +3,16 @@
 /// Font size of the list component, if no size is set.
 /// @group list
 $kendo-list-font-size: null !default;
-$kendo-list-font-size-sm: $font-size-lg !default;
-$kendo-list-font-size-md: $font-size-lg !default;
+$kendo-list-font-size-sm: $font-size-md !default;
+$kendo-list-font-size-md: $font-size-md !default;
 $kendo-list-font-size-lg: $font-size-lg !default;
 
 /// Line height of the list component, if no size is set.
 /// @group list
 $kendo-list-line-height: null !default;
-$kendo-list-line-height-sm: 1.25 !default;
-$kendo-list-line-height-md: 1.25 !default;
-$kendo-list-line-height-lg: 1.25 !default;
+$kendo-list-line-height-sm: (20 / 14) !default;
+$kendo-list-line-height-md: (20 / 14) !default;
+$kendo-list-line-height-lg: 1.5 !default;
```

Comment:
we can use [$line-height-lg](https://github.com/telerik/kendo-themes/blob/develop/packages/material/scss/typography/_variables.scss#L30) here as the values are matching.